src/types/user.ts

```diff
@@ -0,0 +1,12 @@
+export default interface User {
+  user_id: string;
+  email: string;
+  password: string;
+  avatar: string;
+  userName: string;
+  teamName: string;
+  Nationality: string;
+  invitedBy: string | null;
+  isPartialUser: boolean;
+  accessToken: string;
+}
```

Comment:
Don't export default it.

```suggestion
export interface User {
  user_id: string;
  email: string;
  password: string;
  avatar: string;
  userName: string;
  teamName: string;
  Nationality: string;
  invitedBy: string | null;
  isPartialUser: boolean;
  accessToken: string;
}
```

(1) Also, not all the details are required.
avatar and invitedBy, are not required. 
```
  avatar?: string;
  invitedBy?: string;
```

(2) Also, `Nationality` should be lowercase as all.

Reply:
Fixed.